README.md

```diff
@@ -1 +1,92 @@
 # NAK-IAA
+This Project is being developed as an examination for the 
+"Internet Application Architectures" (Internetanwendungsarchitekturen) course of the 
+[Nordakademie](https://www.nordakademie.de) in 2020.
+
+It is being developed by [Til Blechschmidt](https://github.com/TilBlechschmidt), 
+[Noah Peeters](https://github.com/NoahPeeters), [Hendrik Reiter](https://github.com/HenryWedge) 
+and [Hans Rißer](https://github.com/hpr1999). 
+
+The application at hand is a survey tool used to plan various meetings 
+with multiple participants.
+
+We have divided it into the [frontend](/frontend) and the [backend](backend).
+Installation Instructions can be found 
+[here for the frontend](#setting-up-the-frontend) and 
+[here for the backend](#setting-up-the-backend).
+
+
+## Frontend
+
+The frontend of the application was built using [Angular](https://angular.io) 
+and [yarn](https://classic.yarnpkg.com/en/).
+
+### Setting up the Frontend
+
+For a start, you will need 
+[yarn](https://classic.yarnpkg.com/en/docs/install/) and [Angular CLI](https://cli.angular.io).
+
+To make sure everything works right, follow the instructions in [Setting up the API](#setting-up-the-api) first
+and then the ones in [Installing and starting the frontend](#installing-and-starting-the-frontend).
+
+### Setting up the API
+
+This project uses an Open-API 3 specification.
+To set up and generate the API access code in the frontend, clone the repository, 
+open a terminal in its root directory and use the following instructions: 
+
+```shell script
+cd api
+yarn install
+yarn generate
+```
+
+### Installing and starting the frontend
+
+Then, to set up and start the frontend, clone the repository, 
+open a terminal in its root directory and use the following instructions:
+
+```shell script
+cd frontend
+yarn install
+ng serve
+```
+
+## Backend
+
+On the backend, [Spring Boot](https://spring.io/projects/spring-boot) does the heavy lifting.
+The build is done by [Gradle](https://gradle.org).
+
+### Setting up the Backend
+
+You will need a Java15-JDK like 
+[this one](https://adoptopenjdk.net/?variant=openjdk15&jvmVariant=hotspot).
+To set up and start the backend, clone the repository, 
+open a terminal in its root directory and use the following command: 
```

Comment:
We should add that one has to follow the instructions in `#setting-up-the-api` first as well.

Reply:
I moved the API-instructions to the front as its own chapter an referenced it from front- & backend